R/pkg/R/generics.R

```diff
@@ -695,6 +695,12 @@ setGeneric("getField", function(x, ...) { standardGeneric("getField") })
 #' @rdname columnfunctions
 setGeneric("getItem", function(x, ...) { standardGeneric("getItem") })
 
+#' @rdname columnfunctions
+setGeneric("isInf", function(x) { standardGeneric("isInf") })
+
+#' @rdname columnfunctions
+setGeneric("isinf", function(x) { standardGeneric("isinf") })
```

Comment:
`isnan` lower case is not a column functions see https://github.com/NihalHarish/spark/blob/7e396f70f58ffd309e7f738751f3aa8cfe321ce7/R/pkg/R/generics.R#L1002
`@rdname columnfunctions` will cause it to go to the wrong doc page